client/src/GlobalStyles.js

```diff
@@ -37,12 +39,12 @@ h1, h2, h3, h4, h5, h6 {
   text-transform: uppercase;
 }
 h1 {  font-size: 2rem;
-  color: var(--primary-color);
+  color:${(props) => props.theme.h1color};
 }
 h2 {
   font-size: 1.5rem;
-  color: var(--primary-color);
-}
+  color:${(props) => props.theme.h1color};
```

Comment:
Typo?
```suggestion
  color:${(props) => props.theme.h2color};
```

Reply:
changed the name "hcolor" since the colour doesn't just refer to h2 but also to h1 👍